### ISSUE TITLE
Yoast head field for products API

### ISSUE THREAD

**Issue by DonKoko** [co-cart/co-cart#344]
## Describe the bug

After updating our CoCart Lite to the latest version, we noticed that the head tags generated from Yoast were gone from our product pages.

The endpoint we are using to fetch our products is the default Wordpress endpoint for posts. In our case the `post_type` we are fetching is `product`so the endpoint is: `https://website.com/wp-json/wp/v2/product/:id`

By default, Yoast seo adds some fields to the Products api endpoint. More information can be found here: https://developer.yoast.com/customization/apis/rest-api/#1-get-seo-metadata-as-part-of-wordpress-native-wp-rest-api-responses

We usually use the field `yoast_head` to render our head tags, however after the last update CoCart Lite this field disappeared from the response.

We have already confirmed that:

- Disabling CoCart Lite fixes the issue
- The second field generated by Yoast `yoast_head_json` is working correctly
- The issue is present only with the CPT `products`. The issue doesn't happen for other custom post types
- We have the latest version of WooCommerce(Version 6.6.1)
- We have the latest version of Yoast Seo(Version 19.2) & Yoast SEO: WooCommerce(Version 14.6)
- We have the latest version of CoCart Lite(Version 3.7.3) & CoCart - Cart Enhanced(Version 3.2.0)
- Latest version of Wordpress (6.0)

## Prerequisites

- [x] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [ ] The issue still exists against the latest `trunk` branch of CoCart on GitHub.
- [ ] The issue still exists against the latest `dev` branch of CoCart on GitHub (this is **not** the same version as on WordPress.org!).
- [x] I have attempted to find the simplest possible steps to reproduce the issue.
- [ ] I have included a failing test as a pull request (Optional)
- [x] I have installed the requirements to run this plugin.

## Steps to reproduce the issue

1. Update plugins described above to latest version
2. Make a request to the default WP api endpoint for products: https://website.com/wp-json/wp/v2/product/:id
3. Notice that the field `yoast_head` is missing from the response

## Expected/actual behavior

Field shouldn't be missing

## Isolating the problem

- [ ] This bug happens with only WooCommerce and CoCart plugin are active.
- [ ] This bug happens with a default WordPress theme active.
- [ ] This bug happens with the WordPress theme Storefront active.
- [x] This bug happens with the latest release of WooCommerce active.
- [ ] This bug happens only when I authenticate as a customer.
- [ ] This bug happens only when I authenticate as administrator.
- [x] I can reproduce this bug consistently using the steps above.

## WordPress Environment

<details>
```
### WordPress Environment ###

WordPress address (URL): ----
Site address (URL): ----
WC Version: 6.6.1
REST API Version: ✔ 6.6.1
WC Blocks Version: ✔ 7.9.0
Action Scheduler Version: ✔ 3.4.0
Log Directory Writable: ✔
WP Version: 6.0
WP Multisite: –
WP Memory Limit: 2 GB
WP Debug Mode: ✔
WP Cron: ✔
Language: de_DE
External object cache: –

### Server Environment ###

Server Info: Apache/2
PHP Version: 7.4.9
PHP Post Max Size: 1 GB
PHP Time Limit: 300
PHP Max Input Vars: 3000
cURL Version: 7.62.0
OpenSSL/1.0.2k

SUHOSIN Installed: –
MySQL Version: 5.5.5-10.3.23-MariaDB-cll-lve
Max Upload Size: 256 MB
Default Timezone is UTC: ✔
fsockopen/cURL: ✔
SoapClient: ✔
DOMDocument: ✔
GZip: ✔
Multibyte String: ✔
Remote Post: ✔
Remote Get: ✔

### Database ###

WC Database Version: 6.6.1
WC Database Prefix: wp_
Total Database Size: 251.04MB
Database Data Size: 174.11MB
Database Index Size: 76.93MB
wp_woocommerce_sessions: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_woocommerce_api_keys: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_woocommerce_attribute_taxonomies: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_woocommerce_downloadable_product_permissions: Data: 0.02MB + Index: 0.06MB + Engine InnoDB
wp_woocommerce_order_items: Data: 2.52MB + Index: 1.52MB + Engine InnoDB
wp_woocommerce_order_itemmeta: Data: 16.55MB + Index: 13.03MB + Engine InnoDB
wp_woocommerce_tax_rates: Data: 0.02MB + Index: 0.06MB + Engine InnoDB
wp_woocommerce_tax_rate_locations: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_woocommerce_shipping_zones: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_woocommerce_shipping_zone_locations: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_woocommerce_shipping_zone_methods: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_woocommerce_payment_tokens: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_woocommerce_payment_tokenmeta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_woocommerce_log: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_actionscheduler_actions: Data: 1.02MB + Index: 0.20MB + Engine InnoDB
wp_actionscheduler_claims: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_actionscheduler_groups: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_actionscheduler_logs: Data: 1.02MB + Index: 0.14MB + Engine InnoDB
wp_adtribes_my_conversions: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_aws_cache: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_aws_index: Data: 0.48MB + Index: 0.00MB + Engine InnoDB
wp_cocart_carts: Data: 0.05MB + Index: 0.02MB + Engine InnoDB
wp_commentmeta: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
wp_comments: Data: 4.06MB + Index: 1.70MB + Engine MyISAM
wp_elfsight_countdown_timer_widgets: Data: 0.06MB + Index: 0.00MB + Engine InnoDB
wp_fpd_categories: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_fpd_category_products_rel: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_fpd_designs: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_fpd_products: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_fpd_views: Data: 0.05MB + Index: 0.00MB + Engine InnoDB
wp_jtl_connector_category_level: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_jtl_connector_link_category: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_jtl_connector_link_crossselling: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_jtl_connector_link_crossselling_group: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_jtl_connector_link_currency: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_jtl_connector_link_customer: Data: 0.25MB + Index: 0.45MB + Engine InnoDB
wp_jtl_connector_link_customer_group: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_jtl_connector_link_image: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_jtl_connector_link_language: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_jtl_connector_link_manufacturer: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_jtl_connector_link_measurement_unit: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_jtl_connector_link_order: Data: 0.23MB + Index: 0.17MB + Engine InnoDB
wp_jtl_connector_link_payment: Data: 0.23MB + Index: 0.17MB + Engine InnoDB
wp_jtl_connector_link_product: Data: 0.06MB + Index: 0.02MB + Engine InnoDB
wp_jtl_connector_link_shipping_class: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_jtl_connector_link_shipping_method: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_jtl_connector_link_specific: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_jtl_connector_link_specific_value: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_jtl_connector_link_tax_class: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_jtl_connector_product_checksum: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_links: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
wp_litespeed_img_optm: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_mailchimp_carts: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_mailchimp_jobs: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_mollie_pending_payment: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_ms_snippets: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_nextend2_image_storage: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_nextend2_section_storage: Data: 0.02MB + Index: 0.06MB + Engine InnoDB
wp_nextend2_smartslider3_generators: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_nextend2_smartslider3_sliders: Data: 0.05MB + Index: 0.03MB + Engine InnoDB
wp_nextend2_smartslider3_sliders_xref: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_nextend2_smartslider3_slides: Data: 0.05MB + Index: 0.11MB + Engine InnoDB
wp_ngg_album: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_ngg_gallery: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_ngg_pictures: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_options: Data: 45.11MB + Index: 0.22MB + Engine MyISAM
wp_postmeta: Data: 39.49MB + Index: 20.27MB + Engine MyISAM
wp_posts: Data: 14.41MB + Index: 1.39MB + Engine MyISAM
wp_redirection_404: Data: 2.02MB + Index: 0.45MB + Engine InnoDB
wp_redirection_groups: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_redirection_items: Data: 0.02MB + Index: 0.09MB + Engine InnoDB
wp_redirection_logs: Data: 1.25MB + Index: 0.22MB + Engine InnoDB
wp_revslider_css: Data: 0.13MB + Index: 0.00MB + Engine InnoDB
wp_revslider_css_bkp: Data: 0.13MB + Index: 0.00MB + Engine InnoDB
wp_revslider_layer_animations: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_revslider_layer_animations_bkp: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_revslider_navigations: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_revslider_navigations_bkp: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_revslider_sliders: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_revslider_sliders_bkp: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_revslider_slides: Data: 0.14MB + Index: 0.00MB + Engine InnoDB
wp_revslider_slides_bkp: Data: 0.27MB + Index: 0.00MB + Engine InnoDB
wp_revslider_static_slides: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_revslider_static_slides_bkp: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_snippets: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_termmeta: Data: 0.05MB + Index: 0.03MB + Engine InnoDB
wp_terms: Data: 0.00MB + Index: 0.01MB + Engine MyISAM
wp_term_relationships: Data: 0.05MB + Index: 0.10MB + Engine MyISAM
wp_term_taxonomy: Data: 0.01MB + Index: 0.01MB + Engine MyISAM
wp_tm_taskmeta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_tm_tasks: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_usermeta: Data: 0.12MB + Index: 0.05MB + Engine MyISAM
wp_users: Data: 0.00MB + Index: 0.01MB + Engine MyISAM
wp_wc_admin_notes: Data: 0.08MB + Index: 0.00MB + Engine InnoDB
wp_wc_admin_note_actions: Data: 0.05MB + Index: 0.02MB + Engine InnoDB
wp_wc_category_lookup: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wc_customer_lookup: Data: 1.52MB + Index: 0.39MB + Engine InnoDB
wp_wc_download_log: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_wc_order_coupon_lookup: Data: 0.09MB + Index: 0.14MB + Engine InnoDB
wp_wc_order_product_lookup: Data: 1.52MB + Index: 1.50MB + Engine InnoDB
wp_wc_order_stats: Data: 1.52MB + Index: 0.72MB + Engine InnoDB
wp_wc_order_tax_lookup: Data: 0.52MB + Index: 0.45MB + Engine InnoDB
wp_wc_product_attributes_lookup: Data: 0.05MB + Index: 0.02MB + Engine InnoDB
wp_wc_product_download_directories: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_wc_product_meta_lookup: Data: 1.52MB + Index: 2.55MB + Engine InnoDB
wp_wc_rate_limits: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_wc_reserved_stock: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wc_tax_rate_classes: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_wc_webhooks: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_wdr_order_discounts: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wdr_order_item_discounts: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wdr_rules: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_woobe_history: Data: 0.05MB + Index: 0.05MB + Engine InnoDB
wp_woobe_history_bulk: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_woo_mass_discount: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wpfm_backup: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wpmailsmtp_debug_events: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wpmailsmtp_tasks_meta: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wpml_mails: Data: 0.05MB + Index: 0.00MB + Engine InnoDB
wp_wp_quiz_play_data: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_wrc_caches: Data: 0.02MB + Index: 0.05MB + Engine InnoDB
wp_wrc_relations: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_wsal_metadata: Data: 28.02MB + Index: 23.39MB + Engine InnoDB
wp_wsal_occurrences: Data: 1.45MB + Index: 1.77MB + Engine InnoDB
wp_yoast_indexable: Data: 5.52MB + Index: 3.50MB + Engine InnoDB
wp_yoast_indexable_hierarchy: Data: 0.34MB + Index: 0.48MB + Engine InnoDB
wp_yoast_migrations: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_yoast_primary_term: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_yoast_prominent_words: Data: 0.08MB + Index: 0.16MB + Engine InnoDB
wp_yoast_seo_links: Data: 0.14MB + Index: 0.09MB + Engine InnoDB
wp_yoast_seo_meta: Data: 0.11MB + Index: 0.00MB + Engine InnoDB

### Post Type Counts ###

acf-field: 133
acf-field-group: 21
attachment: 1582
components: 22
custom_css: 1
daily-deal-banner: 5
faq_pt: 6
flamingo_contact: 51
flamingo_inbound: 705
ktsprotype: 1
mc4wp-form: 1
mts_notification_bar: 1
nav_menu_item: 71
oembed_cache: 12
page: 35
post: 20
postman_sent_mail: 250
product: 107
product_manuals: 6
product_variation: 2650
revision: 855
shop_coupon: 10
shop_order: 7314
shop_order_refund: 304
urp_review: 3
wp_block: 20
wp_global_styles: 2
wpcf7_contact_form: 5

### Security ###

Secure connection (HTTPS): ✔
Hide errors from visitors: ✔

### Active Plugins (40) ###

Query Monitor: by John Blackbourn – 3.9.0
ACF to REST API: by Aires Gonçalves – 3.3.3
Advanced Custom Fields PRO: by Delicious Brains – 5.12.2
AJAX Thumbnail Rebuild: by junkcoder
ristoniinemets – 1.13

CoCart Lite: by Sébastien Dumont – 3.7.3
CoCart - Cart Enhanced: by Sébastien Dumont – 3.2.0
CoCart Pro: by Sébastien Dumont – 1.0.0-rc.3
Contact Form 7: by Takayuki Miyoshi – 5.5.6.1
Custom Post Type UI: by WebDevStudios – 1.12.1
Elfsight Countdown Timer CC: by Elfsight – 1.4.0
Facebook for WooCommerce: by Facebook – 2.6.16
Flamingo: by Takayuki Miyoshi – 2.2.3
Guillotheme: by David Matthew – 1.0.0
Health Check & Troubleshooting: by Die WordPress.org-Community – 1.4.5
HTTP Headers: by Dimitar Ivanov – 1.18.5
Mollie Payments for WooCommerce: by Mollie – 7.1.0
Images Endpoint for Product Designer: by Whale Agency – 1.0.0
Post SMTP: by Post SMTP – 2.1.1
Post Types Order: by Nsp Code – 1.9.9
Redirection: by John Godley – 5.2.3
Remove Schema: by Plugin.nl – 1.6.1
REST API - Head Tags: by Frontity – 1.2.1
SVG Support: by Benbodhi – 2.4.2
Advanced Editor Tools (previously TinyMCE Advanced): by Automattic – 5.6.0
WebP Konverter für Medien: by Mateusz Gbiorczyk – 4.3.6
Wicked Folders: by Wicked Plugins – 2.18.13
Woo Discount Rules: by Flycart – 2.4.1
WooCommerce Blocks: by Automattic – 7.9.0
Premmerce Permalink Manager for WooCommerce: by premmerce – 2.3.5
Product Feed PRO for WooCommerce: by AdTribes.io – 10.9.4
WooCommerce Bulk Product Editor: by Lagudi Domenico – 2.7
WooCommerce: by Automattic – 6.6.1
WordPress Importer: by wordpressdotorg – 0.7
Yoast SEO Premium: by Team Yoast – 18.0
Yoast SEO: by Team Yoast – 19.2
WP Activity Log for WooCommerce: by WP White Security – 1.4.3
WP-REST-API V2 Menus: by Claudio La Barbera – 0.10
WP REST Filter: by SK8Tech – 1.4.3
WP Activity Log: by WP White Security – 4.4.1
Yoast SEO: WooCommerce: by Team Yoast – 14.6

### Inactive Plugins (20) ###

BEAR – Bulk Editor and Products Manager Professional for WooCommerce: by realmag777 – 1.1.2
Bulk Discounts - WooCommerce Product Category Discounts: by AA-Team – 1.0
Classic Editor: by WordPress Contributors – 1.6.2
CoCart CORS: by Sébastien Dumont – 1.0.1
Custom Layouts - Post + Product grids made easy: by Code Amp – 1.4.6
Disable REST API: by Dave McHale – 1.7
Easy WP SMTP: by wpecommerce
alexanderfoxc – 1.4.7

Fancy Product Designer: by fancyproductdesigner.com – 4.7.3
Fancy Product Designer REST API: by fancyproductdesigner.com – 1.6.4
MetaSlider: by MetaSlider – 3.27.5
Price Commander for WooCommerce: by XforWooCommerce – 1.3.0
Product Blocks for WooCommerce: by GetBowtied – 1.5.1
Product Feed Manager for WooCommerce: by RexTheme – 7.2.7
Trustbadge Reviews for WooCommerce: by vendidero – 4.0.15
WooCommerce Amazon Pay: by WooCommerce – 2.2.1 (update to version 2.2.2 is available)
WooCommerce JTL-Connector: by JTL-Software GmbH – 1.36.1
WP Downgrade | Specific Core Version: by Reisetiger – 1.2.5
WP REST API Controller: by YIKES
Inc. – 2.0.7

WP REST Cache: by Acato – 2021.4.1
WP REST Yoast Meta: by Acato – 2021.1.2

### Must Use Plugins (2) ###

Health Check Troubleshooting Mode: by  – 1.7.2
WordPress automation by Installatron: by  –

### Settings ###

API Enabled: ✔
Force SSL: –
Currency: EUR (€)
Currency Position: right_space
Thousand Separator: .
Decimal Separator: ,
Number of Decimals: 2
Taxonomies: Product Types: external (external)
grouped (grouped)
simple (simple)
variable (variable)

Taxonomies: Product Visibility: exclude-from-catalog (exclude-from-catalog)
exclude-from-search (exclude-from-search)
featured (featured)
outofstock (outofstock)
rated-1 (rated-1)
rated-2 (rated-2)
rated-3 (rated-3)
rated-4 (rated-4)
rated-5 (rated-5)

Connected to WooCommerce.com: –
Enforce Approved Product Download Directories: –

### WC Pages ###

Shop base: #22144 - /shop/
Cart: #22145 - /warenkorb/
Checkout: #22146 - /kasse/
My account: #22147 - /mein-konto/
Terms and conditions: #607 - /agb/

### Theme ###

Name: --
Version: 1.0.0
Author URL: --
Child Theme: ❌ – If you are modifying WooCommerce on a parent theme that you did not build personally we recommend using a child theme. See: How to create a child theme
WooCommerce Support: ✔

### Templates ###

Overrides: –

### Admin ###

Enabled Features: activity-panels
analytics
coupons
customer-effort-score-tracks
experimental-products-task
experimental-import-products-task
experimental-fashion-sample-products
homescreen
marketing
mobile-app-banner
navigation
onboarding
onboarding-tasks
remote-inbox-notifications
remote-free-extensions
payment-gateway-suggestions
shipping-label-banner
subscriptions
store-alerts
transient-notices
wc-pay-promotion
wc-pay-welcome-page
wc-pay-subscriptions-page

Disabled Features: minified-js
settings

Daily Cron: ✔ Next scheduled: 2022-07-06 15:25:36 +01:00
Options: ✔
Notes: 55
Onboarding: completed

### Action Scheduler ###

Complete: 608
Oldest: 2022-06-05 11:23:39 +0000
Newest: 2022-07-06 07:51:45 +0000

Failed: 1
Oldest: 2021-08-16 16:13:26 +0000
Newest: 2021-08-16 16:13:26 +0000

Pending: 1
Oldest: 2022-07-06 09:00:16 +0000
Newest: 2022-07-06 09:00:16 +0000


### CoCart ###

Version: 3.7.3
Database Version: 3.7.3
Install Date: June 24
2020

Carts in Session: 6
Carts Active: 6 out of 6 in session.
Carts Expiring Soon: 3 out of 6 in session.
Carts Expired: 0 out of 6 in session.
Carts Source (by CoCart): 4
Carts Source (by Web): 0
Carts Source (by Other): 2

### Status report information ###

Generated at: 2022-07-06 08:56:42 +01:00
```
</details>

**Comment by DonKoko** [comment]
Some more details. After some more testing we have concluded that 3.4.0 breaks it. In 3.3.0 the API response is as expected with `yoast_head` being present.

**Comment by seb86** [comment]
@DonKoko This is not a bug. It was done intentional as explained in the changelog.

> Keep the JSON response valid and increase performance.

It was causing issues with CoCart so I needed to find a way to un-register the field. WordPress doesn't have that ability by default so I wrote a new function that hopefully I will get round to submitting to the core of WordPress.

Happy to discuss further with any ideas.
